source/query-bar.txt

```diff
@@ -57,7 +57,7 @@ operator.
    Builder to enter a query into the query bar. For details, see
    :ref:`build-query`.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
```

Comment:
I would break the sentence at "disabled" and start with "The Filter label appears red until..." ("turns" implies that it started as black then became red.)